src/panoptes/pocs/camera/camera.py

```diff
@@ -383,7 +383,9 @@ def take_observation(self, observation, headers=None, filename=None, **kwargs):
                 describing the observation
             headers (dict, optional): Header data to be saved along with the file.
```

Comment:
Is this true? It seems like only selected headers are written to the file. We might want to make this clear, and acceptable headers should be documented somewhere.

Reply:
I've created #1013 to address this, but yes, it's an annoying problem.